tests/functional/pv/pv_services/test_cr_resources_validation.py

```diff
@@ -22,10 +22,10 @@
 ERRMSG = "Error in command"
 
 
-@magenta_squad
-@tier3
-@skipif_ocp_version("<4.13")
-@skipif_ocs_version("<4.13")
+@green_squad
+@tier2
+@skipif_ocp_version("<4.15")
```

Comment:
this won't be needed anymore as we have release branches now.

Reply:
@skipif_ocp_version("<4.15")   - should I remove this row only?   @keesturam

Reply:
or also I should remove @skipif_ocs_version("<4.15")  ?  @keesturam

Reply:
@keesturam I've removed @skipif_ocs_version("<4.15") and left @skipif_ocp_version("<4.15") since we support ocp (n-1) vesion ( previous version of ocp with current odf).